sklearn/ensemble/_stacking.py

```diff
@@ -74,6 +84,11 @@ def _concatenate_predictions(self, predictions):
                     X_meta.append(preds[:, 1:])
                 else:
                     X_meta.append(preds)
+        if self.passthrough:
+            X_meta.append(X)
+            if issparse(X):
+                return sparse_hstack(X_meta).tocsr()
```

Comment:
Nit: It would most likely be clearer for future devs:

```py
import scipy.sparse as sparse

...
if self.passthrough:
    X_meta.append(X)

if sparse.issparse(X):
    return sparse.hstack(...
return np.concatenate(X_meta, axis=1)
```